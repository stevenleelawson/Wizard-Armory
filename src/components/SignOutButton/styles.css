.nav-btn {
  font-family: 'Lora', serif;
  border: none;
  font-size: 1rem;
}
