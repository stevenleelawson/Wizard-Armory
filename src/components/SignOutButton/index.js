import React from 'react';
import './styles.css';

import { auth } from '../../firebase';

const SignOutButton = () =>
  <button
    type='button'
    onClick={auth.doSignOut}
    className='nav-btn'
  >
    Sign Out
  </button>;

export default SignOutButton;
