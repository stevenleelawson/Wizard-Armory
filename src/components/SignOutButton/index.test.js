import React from 'react';
import { shallow } from 'enzyme';
import SignOutButton from './index';

describe('SignOutButton', () => {
  it('should match the snapshot', () => {
    const wrapper = shallow(<SignOutButton />);
    expect(wrapper).toMatchSnapshot();
  });
});
