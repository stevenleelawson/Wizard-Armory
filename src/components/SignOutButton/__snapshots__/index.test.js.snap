// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SignOutButton should match the snapshot 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <SignOutButton />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": "Sign Out",
      "className": "nav-btn",
      "onClick": [Function],
      "type": "button",
    },
    "ref": null,
    "rendered": "Sign Out",
    "type": "button",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": "Sign Out",
        "className": "nav-btn",
        "onClick": [Function],
        "type": "button",
      },
      "ref": null,
      "rendered": "Sign Out",
      "type": "button",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
